community/kernel/src/main/java/org/neo4j/kernel/impl/api/DataOperationsFacade.java

```diff
@@ -92,14 +96,15 @@
 import org.neo4j.storageengine.api.lock.ResourceType;
 import org.neo4j.storageengine.api.schema.PopulationProgress;
 
-public class OperationsFacade implements ReadOperations, DataWriteOperations, SchemaWriteOperations
+public class DataOperationsFacade
```

Comment:
i find new name a bit more confusing then before because at least newly introduced QueryRegistryOperations do not sounds like have something to do with `Data`